x/genutil/v2/cli/export.go

```
@@ -105,6 +110,8 @@
 		StringSlice(flagJailAllowedAddrs, []string{}, "Comma-separated list of operator addresses of jailed validators to unjail")
 	cmd.Flags().
 		String(flags.FlagOutputDocument, "", "Exported state is written to the given file instead of STDOUT")
+	cmd.Flags().Bool(flagForZeroHeight, false, "Export state to start at height zero. Equivalent to --height=0")
+	cmd.Flags().MarkDeprecated(flagForZeroHeight, "use --height=0 instead")
```

golangci-lint (1.62.2)

Error return value of `(*github.com/spf13/pflag.FlagSet).MarkDeprecated` is not checked

(errcheck)